src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Runtime/BasicInstantiateArgumentExceptionsCorrectly.Fixer.vb

```diff
@@ -13,5 +13,8 @@ Namespace Microsoft.NetCore.VisualBasic.Analyzers.Runtime
     Public NotInheritable Class BasicInstantiateArgumentExceptionsCorrectlyFixer
         Inherits InstantiateArgumentExceptionsCorrectlyFixer
 
+        Protected Overrides Sub PopulateCodeFix(context As CodeFixContext, diagnostic As Diagnostic, paramPositionString As String, node As SyntaxNode)
+            Throw New NotImplementedException()
```

Comment:
I assume this is not an acceptable fixer :smile:.

Since the type is sealed (NotInheritable) and doesn't do anything; why do we even build it currently?

If it can go to IOperation I expect we're deleting it; but if we have to play with syntax nodes and we're not making it work I assume it's better to just delete it now, too.